src/main/java/com/mraof/minestuck/data/recipe/MinestuckRecipeProvider.java

```diff
@@ -167,12 +167,16 @@ protected void buildRecipes(RecipeOutput recipeBuilder)
 		ShapedRecipeBuilder.shaped(RecipeCategory.BUILDING_BLOCKS, MSBlocks.END_BOOKSHELF.get()).define('b', Items.BOOK).define('p', MSBlocks.END_PLANKS.get()).pattern("ppp").pattern("bpb").pattern("ppp").group("bookshelf").unlockedBy("has_planks", has(MSBlocks.END_PLANKS.get())).save(recipeBuilder);
 		ShapedRecipeBuilder.shaped(RecipeCategory.BUILDING_BLOCKS, MSBlocks.DEAD_BOOKSHELF.get()).define('b', Items.BOOK).define('p', MSBlocks.DEAD_PLANKS.get()).pattern("ppp").pattern("bpb").pattern("ppp").group("bookshelf").unlockedBy("has_planks", has(MSBlocks.DEAD_PLANKS.get())).save(recipeBuilder);
 		ShapedRecipeBuilder.shaped(RecipeCategory.BUILDING_BLOCKS, MSBlocks.TREATED_BOOKSHELF.get()).define('b', Items.BOOK).define('p', MSBlocks.TREATED_PLANKS.get()).pattern("ppp").pattern("bpb").pattern("ppp").group("bookshelf").unlockedBy("has_planks", has(MSBlocks.TREATED_PLANKS.get())).save(recipeBuilder);
+		ShapedRecipeBuilder.shaped(RecipeCategory.BUILDING_BLOCKS, MSBlocks.SHADEWOOD_BOOKSHELF.get()).define('b', Items.BOOK).define('p', MSBlocks.SHADEWOOD_PLANKS.get()).pattern("ppp").pattern("bpb").pattern("ppp").group("bookshelf").unlockedBy("has_planks", has(MSBlocks.SHADEWOOD_PLANKS.get())).save(recipeBuilder);
+		ShapedRecipeBuilder.shaped(RecipeCategory.BUILDING_BLOCKS, MSBlocks.CINDERED_BOOKSHELF.get()).define('b', Items.BOOK).define('p', MSBlocks.CINDERED_PLANKS.get()).pattern("ppp").pattern("bpb").pattern("ppp").group("bookshelf").unlockedBy("has_planks", has(MSBlocks.CINDERED_PLANKS.get())).save(recipeBuilder);
```

Comment:
@Dweblenod mind looking at this? It seems that only a subset of bookshelf recipes was changed in your PR.

Reply:
While my name is not Dweb, I'm guessing they just never got updated to use the CommonRecipes instead of ShapedRecipeBuilder. I can work that but would prefer to do it as a separate PR.

Reply:
Sure, feel free to do so.